native/app.py

```
@@ -0,0 +1,154 @@
+GROUPS_DIR = DATA_DIR / "groups"
+CONFIGS_DIR = DATA_DIR / "configs"
+MCPJUNGLE_BIN = os.getenv("MCPJUNGLE_BIN", "/usr/local/bin/mcpjungle")
+PORT = int(os.getenv("EMCP_MANAGER_PORT", os.getenv("PORT", "3701")))
+
+app = Flask(__name__, static_folder=None)
+
+
+def mcpj(*args, timeout=180):
+    """Run mcpjungle CLI; long timeout because stdio MCP servers may be slow to init."""
+    cmd = [MCPJUNGLE_BIN, "--registry", GATEWAY_URL, *args]
+    return subprocess.run(cmd, capture_output=True, text=True, timeout=timeout)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)